2-WebApp-graph-user/2-5-HybridFlow/wwwroot/js/RedeemCode.js

```diff
@@ -0,0 +1,27 @@
+// IIFE to acquire token from auth code sent through server.
+(function () {
+    const code = document.cookie.split(';').reduce((cookies, cookie) => {
+        const [name, value] = cookie.split('=').map(c => c.trim());
+        cookies[name] = value;
+        return cookies;
+    }, {})['Microsoft.Identity.Hybrid.Authentication.Cookie'];
```

Comment:
As noted earlier, prefer to have this JS rendered with templating and then have the code written directly to a JS variable.

Reply:
Done.

See `_Layout.cshtml`